src/libsync/bandwidthmanager.cpp

```diff
@@ -373,17 +375,17 @@ void BandwidthManager::switchingTimerExpired()
 
 void BandwidthManager::absoluteLimitTimerExpired()
 {
-    if (usingAbsoluteUploadLimit() && _absoluteUploadDeviceList.count() > 0) {
-        qint64 quotaPerDevice = _currentUploadLimit / qMax(1, _absoluteUploadDeviceList.count());
-        qCDebug(lcBandwidthManager) << quotaPerDevice << _absoluteUploadDeviceList.count() << _currentUploadLimit;
+    if (usingAbsoluteUploadLimit() && _absoluteUploadDeviceList.size() > 0) {
```

Comment:
Another opportunity for: `&& !_absoluteUploadDeviceList.empty()` 😊

Reply:
Done with d6f9bd00174af3eb7ee05ac72a7e3695a351a548.